slugify.js

```diff
@@ -53,6 +53,12 @@
         .replace(new RegExp('[\\s' + replacement + ']+', 'g'), replacement)
     }
 
+    if (options.trim) {
+      slug = slug
+        .replace(new RegExp('^' + replacement + '+'), '')
```

Comment:
This will have unexpected results in some situations. For example:

* If `replacement` is `.`, this will cause `slug` to always be an empty string.
* If `replacement` is `+`, this will cause a `SyntaxError`.